<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить фильм</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="movieTitle">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Описание:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="movieOverview"/>
   </item>
   <item>
    <widget class="QPushButton" name="choosePosterButton">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Выбрать постер</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="posterPreview"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить карточку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
